lib/stripe/api.ex

```diff
@@ -17,15 +17,15 @@ defmodule Stripe.API do
 
   @pool_name __MODULE__
   @api_version "2018-08-23"
-  @http_module Application.get_env(:stripity_stripe, :http_module) || :hackney
 
   @doc """
   In config.exs your implicit or expicit configuration is:
     config :stripity_stripe,
       json_library: Jason # defaults to Poison but can be configured to Jason
   """
+  @spec json_library() :: module
```

Comment:
I think in general we can re-evaluate some of these as global options.  Made an issue here.  For now this PR is g2g though!

https://github.com/code-corps/stripity_stripe/issues/448